invoice/tests.py

```diff
@@ -598,6 +634,32 @@ def test_due_date_equal_date(self):
                                          customer=Customer.objects.first(), date=date, due_date=due_date)
         invoice.full_clean()
 
+    @given(lists(build_invoice_item(), min_size=1, max_size=100))
+    def test_correct_sum(self, invoice_items):
+        date = now()
+        due_date = date
+        invoice = Invoice.objects.create(invoice_number=1, vendor=Vendor.objects.first(),
+                                         customer=Customer.objects.first(), date=date, due_date=due_date)
+        for item in invoice_items:
+            item.invoice = invoice
+            item.save()
+        net_total = sum(item.net_total for item in invoice_items)
+        net_total_string = f'{net_total:.2f} EUR'
+        vat_total = sum(item.tax_amount for item in invoice_items)
+        vat_total_string = f'{vat_total:.2f} EUR'
+        total_total = sum(item.total for item in invoice_items)
+        total_total_string = f'{total_total:.2f} EUR'
+        manual_total = net_total + vat_total
```

Comment:
this is not rounded yet I think.
We want to check of the net total that is visible in the UI (two digits after comma) plus the tax amount that is visible in the UI (two digits after comma) equal the visible total (two digits after comma) exactly.
You are adding the exact values here and then rounding after.

Reply:
Isn't that the lines 653 & 661?

Reply:
no, i updated the test